corehq/apps/app_manager/models.py

```diff
@@ -5311,6 +5369,13 @@ def fetch_file(self, location):
 
         return location, content
 
+    def get_build_langs(self):
+        if self.build_profiles:
+            # return first profile, generated as part of lazy migration
+            return self.build_profiles[self.build_profiles.keys()[0]].langs
```

Comment:
RemoteApps arent really supported for this feature, but some used the existing feature this replaces. As a compromise, we are supporting remote apps with only 1 build profile for legacy reasons, and so this should be deterministic for those. Problems should only occur if they have more than 1